### ISSUE TITLE
Variable substitution fails for task name when there are two variables

### ISSUE THREAD

**Issue by ultraserve-dcheung** [ansible/ansible#5609]
When you have two variables in a task name, it stops performing variable substitution. This only occurs in the output of the playbook. The actual task is fine. 

This works:

```
- name: "Fetch a copy of {{ github_repo }}"
  git: repo=ssh://... version={{ github_version }}
```

Running the playbook gives:

```
TASK: [github-repo | Fetch a copy of example_repo ] **********************************
```

This does not work:

```
- name: "Fetch version {{ github_version }} of {{ github_repo }}"
  git: repo=ssh://... version={{ github_version }}
```

Running the playbook gives:

```
TASK: [github-repo | Fetch version {{ github_version }} of {{ github_repo }}] **********
```

**Comment by mpdehaan** [comment]
This is not the case, most likely one variable is an inventory scoped variable.

Please stop by the mailing list if you have questions.

Thanks!

**Comment by ultraserve-dcheung** [comment]
All variables are defined at the role level.

On Wednesday, 15 January 2014, Michael DeHaan wrote:

> Closed #5609 https://github.com/ansible/ansible/issues/5609.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ansible/ansible/issues/5609
> .

## 

Regards,

_Danny Cheung  *|  IaaS Engineer
*Ultra Serve Internet Pty Ltd_

Mobile: 0416 729 199
Email: dcheung@ultraserve.com.au

Phone : +61 2 9563 4800  |  Fax : +61 2 9563 4899
_1800 467 863  | *_www.ultraserve.com.au http://www.ultraserve.com.au/*

## 

This electronic message together with any attachments is intended to be 
read by the named recipient only. It may contain information that is 
confidential, proprietary or the subject of legal privilege. If you are not 
the intended recipient, you may not use, disclose or copy  any of the 
information in this email. If you have received this email in error, please 
reply immediately and delete this e-mail. Thank you.
